changelogs/fragments/9432-deprecate-pure.yml

```diff
@@ -0,0 +1,3 @@
+deprecated_features:
+  - pure module_utils - modules using this were removed in community.general 3.0.0 (https://github.com/ansible-collections/community.general/pull/9432).
```

Comment:
I guess this should now be changed to be about the doc fragment.

Reply:
You lost me. The mod utils is still being deprecated, that has not changed, but #9433 has just removed the doc fragment, so how should this PR be about the doc fragment that no longer exists?

Reply:
The file plugins/doc_fragments/purestorage.py is still there, so it has not been removed and has to be deprecated before we can delete it.

Reply:
(The only thing that has been removed is the DOCUMENTATION fragment in plugins/doc_fragments/purestorage.py. The other fragments in there are still around.)